RecoParticleFlow/PFClusterProducer/src/SpikeAndDoubleSpikeCleaner.cc

```diff
@@ -117,10 +117,55 @@ clean(const edm::Handle<reco::PFRecHitCollection>& input,
     }    
     const spike_cleaning& clean = _thresholds.find(hitlayer)->second;    
     if( rechit.energy() < clean._singleSpikeThresh ) continue;
+
+
+
+    //Fix needed for HF
+    float compsumE = 0.0;
+    if ((hitlayer==12 or hitlayer==11))
```

Comment:
This says to only apply the fix only to the HF layers.  I think this should be hard coded, as any other values would be buggy